app/_src/policies/meshloadbalancingstrategy.md

```diff
@@ -237,6 +278,153 @@ Apply the configuration with `kumactl apply -f [..]` or with the [HTTP API](/doc
 {% endtab %}
 {% endtabs %}
 
+{% if_version gte:2.5.x %}
+### Disable cross zone traffic and prioritize traffic the dataplanes on the same node and availability zone
+
+In this example, whenever a user sends a request to the `backend` service, 90% of the requests will arrive at the instance with the same value of the `k8s.io/node` tag, 9% of the requests will go to the instance with the same value as the caller of the `k8s.io/az` tag, and 1% will go to the rest of the instances.
+
+{% policy_yaml local-zone-affinity-backend %}
+```yaml
+type: MeshLoadBalancingStrategy
+name: local-zone-affinity-backend
+mesh: mesh-1
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+      default:
+        localityAwareness:
+          localZone:
+            affinityTags:
+              - key: k8s.io/node
+              - key: k8s.io/az
+```
+{% endpolicy_yaml %}
+
+### Disable cross zone traffic and route to the local zone instances equally
+
+In this example, when a user sends a request to the backend service, the request is routed equally to all instances in the local zone. If there are no instances in the local zone, the request will fail because there is no cross-zone traffic.
+
+{% policy_yaml local-zone-affinity-backend-2 %}
+```yaml
+type: MeshLoadBalancingStrategy
+name: local-zone-affinity-backend
+mesh: mesh-1
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+      default:
+        localityAwareness:
+          localZone:
+            affinityTags: []
+```
+{% endpolicy_yaml %}
+
+or 
+
+{% policy_yaml local-zone-affinity-backend-3 %}
+```yaml
+type: MeshLoadBalancingStrategy
+name: local-zone-affinity-backend
+mesh: mesh-1
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+      default:
+        localityAwareness:
+          localZone: {}
```

Comment:
Is this the same as not having a policy altogether?

Reply:
No, it's like it works for `disabled`. If there is 
```
type: MeshLoadBalancingStrategy
name: local-zone-affinity-backend
mesh: mesh-1
spec:
  targetRef:
    kind: Mesh
  to:
    - targetRef:
        kind: MeshService
        name: backend
      default:
        localityAwareness: {}```

then `disabled: false`  and it's configured cross zone traffic with priority 1. We are checking if `localZone` or `crossZone` is configured to. configure the policy, which can be and empty object.

Reply:
Ok so this contradicts the at the top of this doc:

> Locality-aware load balancing is enabled by default unlike its predecessor [localityAwareLoadBalancing](https://deploy-preview-1506--kuma.netlify.app/docs/2.5.x/policies/locality-aware).

Reply:
Btw: 

> When disabled requests are distributed across all endpoints regardless of locality

Is this still a thing? What does "disabled: true" do now?